README.md

```diff
@@ -3,9 +3,33 @@
 
 [![Build Status](https://travis-ci.org/hmrc/openid-connect-userinfo.svg?branch=master)](https://travis-ci.org/hmrc/openid-connect-userinfo) [ ![Download](https://api.bintray.com/packages/hmrc/releases/openid-connect-userinfo/images/download.svg) ](https://bintray.com/hmrc/releases/openid-connect-userinfo/_latestVersion)
 
-This is a placeholder README.md for a new repository
+This REST API provides a specification compliant OpenID Connect implementation. It allows consumers to access user details with consent and in the OpenID Connect UserInfo format. 
```

Comment:
Until some subsequent tickets are played, we're not 100% spec compliant.  The missing part is returning a `sub` field in our userinfo payload.

Perhaps we can add a 'Beta' or 'Alpha' flag (or similar) to indicate that the service is still under development, striving for specification compliance?